www/main/toolbar.js

```diff
@@ -281,51 +288,54 @@ const adminContent = h('div', [
   ),
 ]);
 
-const syncContent = h('div.sync-content', [
-  h('div.sync-code-label', i18n.t('TOOLBAR.SYNC_CONTROLLER.UNIQUE_CODE_LABEL')),
-  h('div.sync-code-num', code),
-  syncItemFactory(
-    i18n.t('TOOLBAR.SYNC_CONTROLLER.SANGAT_SYNC'),
-    h('span', [
-      i18n.t('TOOLBAR.SYNC_CONTROLLER.SYNC_DESC.1'),
-      h('strong', 'sttm.co/sync'),
-      i18n.t('TOOLBAR.SYNC_CONTROLLER.SYNC_DESC.2'),
-    ]),
-    h(
-      'button.button.copy-code-btn',
-      {
-        onclick: () => {
-          if (code !== '...') {
-            const syncString = i18n.t('TOOLBAR.SYNC_CONTROLLER.SYNC_STRING', { code });
-            global.controller.sendText(syncString);
-            analytics.trackEvent('controller', 'codePresented', true);
-          }
+const syncContent = h('div.sync-content-wrapper', [
```

Comment:
would have been cool to do this as a react component.

Reply:
Yes, but moving the whole thing to components would require somewhat bigger refactor that I am tracking in a separate ticket / branch. We wanted to do this in current release, so I just switched back to dev branch and fixed this first.